six/common/aggregator.c

```diff
@@ -229,30 +240,42 @@ static PyObject *submit_event(PyObject *self, PyObject *args) {
         return NULL;
     }
 
-    // notice: PyDict_GetItemString returns a borrowed ref
+    // notice: PyDict_GetItemString returns a borrowed ref or NULL if key was not found
     ev->title = as_string(PyDict_GetItemString(event_dict, "msg_title"));
     ev->text = as_string(PyDict_GetItemString(event_dict, "msg_text"));
-    ev->ts = PyLong_AsLong(PyDict_GetItemString(event_dict, "timestamp"));
+    // PyLong_AsLong will fail if called passing a NULL argument, be safe
```

Comment:
this was hidden because `py_tag_to_c` called with no tags caused an early return